examples/fluids/include/ceed-utils.h

```diff
@@ -1,5 +1,10 @@
-#ifndef CEED_UTILS_H
-#define CEED_UTILS_H
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and other CEED contributors.
```

Comment:
```suggestion
// Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and other CEED contributors.
```

Similar to the other license headers as well.